proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md

```diff
@@ -0,0 +1,58 @@
+# JavaScriptLandia - Phase 1 - Spec
+## October 2020 
+
+### Summary
+JavaScriptLandia is a place where all JavaScript developers, whether if you're all about TypeScript, a React evangelist, or you stick to Node.js, gather to declare their support for the JavaScript ecosystem and OpenJS. 
+
+### Phase 1 Components
+
+**Launch Announcement
+
+This will be a blog post on the OpenJS Site
+Developed in partnership with the content team at the LF 
```

Comment:
Curious why the LF is involved, here, or is this just OpenJSF staff?

Reply:
The OpenJS staff are actually LF staff contracted to us... so this is primarily semantics

Reply:
Well, whether this effort is an OpenJSF effort or a LF/OpenJSF effort does have different implications. It wasn’t clear to me from reading this which one it was. I think it’s worth clarifying.